example/fittemplate.md

```diff
@@ -0,0 +1,157 @@
+[MATLAB path](/faq/should_i_add_fieldtrip_with_all_subdirectories_to_my_matlab_path) properly.
+
+You can load the head shape measured during the MEG recording with the Polhemus and a
+template volume conduction model. We have to ensure that they have consistent units, hence
+we will convert the units into mm.
+
+    polhemus = ft_read_headshape('epilepsy/case1/ctf_data/case1.pos');
+    polhemus = ft_convert_units(polhemus,'mm');
+
+    template = ft_read_vol('standard_bem.mat');
+    template = ft_convert_units(template,'mm');
+
+## Coregistration
+
+In the next step we coregister the template head model with the Polhemus head shape to ensure that they are expressed consistently in the MEG coordinate system (i.e. relative to the same origin and with the axes pointing in the same direction). The coregistration consists of a rotation and translation.
+
+    cfg = [];
+    cfg.template.headshape      = polhemus;
+    cfg.checksize               = inf;
+    cfg.individual.headmodel    = template;
+    cfg                         = ft_interactiverealign(cfg);
+    template                    = ft_transform_geometry(cfg.m, template);
+
+## Removing unshared details between Polhemus and the template
+
+To create the individualized head model it is important that the scalp
+surface of the template head model only contains features that are also in the Polhemus head
+shape, and vice versa. We can use **[ft_defacemesh](/reference/ft_defacemesh)** to remove some features.
+
+So first we visualise both meshes:
+
+    ft_plot_mesh(template.bnd);  
+    ft_plot_mesh(polhemus);  
+
+{% include image src="/assets/img/example/fittemplate/template_surface_polhemus_surface_plot.png" %}
+
+As the template model and the Polhemus measurement have exclusive features, we will now remove these  
+
+    defaced_template      = template.bnd(1);
+    defaced_template.unit = template.unit
+
+    cfg              = [];
+    cfg.translate    = [-40 0 -50];
+    cfg.scale        = [200 200 200];
+    cfg.rotate       = [0 0 0];
+    defaced_template =  ft_defacemesh(cfg, defaced_template);
+
+    cfg              = [];
+    cfg.translate    = [-40 0 -50];
+    cfg.scale        = [200 200 200];
+    cfg.rotate       = [0 0 0];
+    defaced_polhemus =  ft_defacemesh(cfg, polhemus);
+
+Now we have another look at the surfaces
+
+    ft_plot_mesh(defaced_template);  
+    ft_plot_mesh(defaced_polhemus);  
+
+{% include image src="/assets/img/example/fittemplate/defaced_template_surface_polhemus_surface_plot.png" %}
+
+
+## Determining and applying of the transformation
+
+### Method 1: On the basis of spheres fitted to the head shapes
+
+For the first approach we will fit a sphere to the three fiducials and we will fit another sphere to the template head model. On the basis of difference of the center and scaling between the two spheres we can derive a transformation containing information of the translation and the global scaling.
```

Comment:
you don't fit a sphere to the fiducials

Reply:
you could describe another "method" that scales it on basis of the fiducials from the individual and the template. That woudl be useful if there is no headshape (as e.g. in the case where no Polhemus were used).

Reply:
> you don't fit a sphere to the fiducials

correct this.

Reply:
> you could describe another "method" that scales it on basis of the fiducials from the individual and the template. That woudl be useful if there is no headshape (as e.g. in the case where no Polhemus were used).

Implemented this method